lib/browser/api/web-contents.js

```diff
@@ -120,7 +130,18 @@ const asyncWebFrameMethods = function (requestId, method, callback, ...args) {
         if (typeof callback === 'function') callback(result)
         resolve(result)
       } else {
-        reject(error)
+        if (error && error.__ELECTRON_SERIALIZED_ERROR__) {
```

Comment:
`error` is guaranteed to be not `null` nor `undefined` here, do you really need `if (error &&`?